### PR TITLE
Various quest improvements/additions

### PR DESCRIPTION
**Pull request by pm065**
Includes an undergarden quest in Adventure, subtitles for all the chapters and many more quests (notably the gating quests) and an iron jetpacks questline in Tools. This probably has a bunch of mistakes, since it's quite a large pull request.

Images:
https://gyazo.com/5b9d2313f19698beafefe960645f0550
https://gyazo.com/7a32a7f506fba01e952641cb8364edf2

### CODE REVIEW

**Comment by NillerMedDild** [comment]
Thanks for this, I look forward to checking it out! Could you please base the PR on the Develop branch? :)

**Comment by pm065** [comment]
Ah, sorry! One second...

**Comment by pm065** [comment]
Wait, do you mean pushing it there instead of master? I'm pretty new to GitHub 🤔

**Comment by NillerMedDild** [comment]
Oh I read wrong, you did this right, my bad 😄

File: config/ftbquests/quests/chapters/gates_wip.snbt [hunk 1]
```diff
@@ -261,6 +279,7 @@
 			title: "Industrial Foregoing",
 			x: -3.0d,
 			y: 4.5d,
+			subtitle: "ELSE: you don't need to automate all the things",
```
Comment (NillerMedDild) [inline config/ftbquests/quests/chapters/gates_wip.snbt:282]: Not your best subtitle this if/else, got any other ideas? :x

File: config/ftbquests/quests/chapters/industrial_foregoing.snbt [hunk 2]
```diff
@@ -5,6 +5,7 @@
 	filename: "industrial_foregoing",
 	title: "Industrial Foregoing",
 	icon: "industrialforegoing:latex_processing_unit",
+	subtitle: ["IF (pink_slime) then {industrial_foregoing();}"],
```
Comment (NillerMedDild) [inline config/ftbquests/quests/chapters/industrial_foregoing.snbt:8]: Second occurrence of if/else subtitle, same deal here.

File: config/ftbquests/quests/chapters/pedestals.snbt [hunk 3]
```diff
@@ -1437,8 +1438,8 @@
 			}]
 		},
 		{
-			x: 2.5d,
-			y: -2.5d,
+			x: 1.5d,
+			y: 0.0d,
```
Comment (NillerMedDild) [inline config/ftbquests/quests/chapters/pedestals.snbt:1442]: How come you're moving things around in this chapter? :)

**Review by NillerMedDild — CHANGES_REQUESTED** [review]
I liked the vast majority of this PR, gj and thank you :D I would like the below changes though.

**Comment by NillerMedDild** [comment]
I'll fix the conflict when the time comes btw :)

File: config/ftbquests/quests/chapters/expert__bragging_rights.snbt [hunk 4]
```diff
@@ -144,14 +145,44 @@
...
+		},
+		{
+			x: -1.0d,
+			y: -2.0d,
+			id: "3041E4BA0BAA9240",
+			tasks: [{
+				id: "67B0B83C507BF9D9",
+				type: "item",
+				item: "refinedstorage:creative_storage_disk"
+			}]
```
[the remark this answers is not in the document]
Comment (pm065) [reply config/ftbquests/quests/chapters/expert__bragging_rights.snbt:185]: okey

File: config/ftbquests/quests/chapters/botania.snbt [hunk 5]
```diff
@@ -5,6 +5,7 @@
 	filename: "botania",
 	title: "Botania",
 	icon: "botania:mana_pylon",
+	subtitle: ["Every time someone calls Botania a magic mod, Vazkii lets out a Psi"],
```
[the remark this answers is not in the document]
Comment (pm065) [reply config/ftbquests/quests/chapters/botania.snbt:8]: good point

File: config/ftbquests/quests/chapters/astral_sorcery_wip.snbt [hunk 6]
```diff
@@ -5,6 +5,7 @@
 	filename: "astral_sorcery_wip",
 	title: "Astral Sorcery",
 	icon: "astralsorcery:hand_telescope",
+	subtitle: ["Mmm, star sauce"],
```
[the remark this answers is not in the document]
Comment (pm065) [reply config/ftbquests/quests/chapters/astral_sorcery_wip.snbt:8]: changing

File: config/ftbquests/quests/chapters/gates_wip.snbt [hunk 7]
```diff
@@ -5,15 +5,17 @@
 	filename: "gates_wip",
 	title: "Gates",
 	icon: "mapperbase:steel_fence_gate",
+	subtitle: ["The master progression chapter"],
 	always_invisible: true,
 	default_quest_shape: "",
 	quests: [
 		{
-			title: "The Experts 2: Electric Boogalo",
+			title: "Expert 2: Electric Boogaloo",
```
[the remark this answers is not in the document]
Comment (pm065) [reply config/ftbquests/quests/chapters/gates_wip.snbt:13]: oh right, didn't know about that

File: config/ftbquests/quests/chapters/gates_wip.snbt [hunk 8]
```diff
@@ -5,15 +5,17 @@
 	filename: "gates_wip",
 	title: "Gates",
 	icon: "mapperbase:steel_fence_gate",
+	subtitle: ["The master progression chapter"],
```
[the remark this answers is not in the document]
Comment (pm065) [reply config/ftbquests/quests/chapters/gates_wip.snbt:8]: 🤔

**Comment by pm065** [comment]
All very good points, I'll make the changes right away

**Comment by pm065** [comment]
Problems unproblemified I think

**Review by NillerMedDild — APPROVED** [review]

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Awesome, gj :D

**Pull request merged**